src/api/app/models/bs_request_action.rb

```diff
@@ -824,6 +818,10 @@ def set_sourceupdate_default(user)
     update(sourceupdate: 'cleanup') if target_project && user.branch_project_name(target_project) == source_project
   end
 
+  def uniq_key
+    ' '
```

Comment:
```suggestion
    raise AbstractMethodCalled
```

Reply:
I don't think this is a good idea. This runs in a `before_validation`, so basically on instantiation. 

Pushing this out to individual `BsRequestAction` validations would be a ton of work I would like to avoid here.

Reply:
alright